device.h

```diff
@@ -127,6 +128,7 @@ enum {
 	DEV_OPT_ARP_ACCEPT		= (1ULL << 29),
 	DEV_OPT_SPEED			= (1ULL << 30),
 	DEV_OPT_DUPLEX			= (1ULL << 31),
+	DEV_OPT_EEE			= (1ULL << 33),
```

Comment:
Why was bit 32 skipped here?

Reply:
thanks for pointing this out, I have a custom option here which I will probably upstream as well shortly which occupied 32 and I overlooked that when pushing this one. Update to 32 and then other option can become 33 when I push it.